application/src/sass/application.scss

```diff
@@ -3,6 +3,9 @@ $govuk-compatibility-govukfrontendtoolkit: true;
 $govuk-compatibility-govuktemplate: true;
 $govuk-compatibility-govukelements: true;
 
+/* Set the GOV.UK Frontend assets path */
```

Comment:
MIght it be worth adding to the comment that it will look in `/images` and `/fonts` below this directory?

Reply:
👍